src/python/txtai/cloud/storage.py

```diff
@@ -45,15 +48,16 @@ def __init__(self, config):
         # Get driver for provider
         driver = get_driver(config["provider"])
 
+        # A key is required to set by libcloud, but not actually required for implicit auth with a service account
+        key = os.environ.get("ACCESS_KEY") if os.environ.get("ACCESS_KEY") else "default-key-for-implicit-auth"
+        if key is None:
```

Comment:
I would remove this warning. Not all providers require an ACCESS_KEY. For example the local provider doesn't.

Reply:
Sounds good. I don't have a strong opinion on this, and I'll defer to you on it.